bridge/tracker/rootchain.go

```diff
@@ -0,0 +1,236 @@
+package tracker
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/0xPolygon/polygon-edge/blockchain/storage"
+	"github.com/0xPolygon/polygon-edge/blockchain/storage/leveldb"
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/hashicorp/go-hclog"
+	"github.com/umbracle/go-web3"
+	"github.com/umbracle/go-web3/abi"
+	client "github.com/umbracle/go-web3/jsonrpc"
+	"math/big"
+	"os"
+	"path/filepath"
+)
+
+const (
+	//	required block depth for fetching events on the rootchain
+	DefaultBlockConfirmations = 6
+
+	//	Ropsten testnet
+	//rootchainWS   = "wss://ropsten.infura.io/ws/v3/17eac086ff36442ebd43737400eb71ca"
+
+	//	Polygon Edge
+	rootchainWS = "ws://127.0.0.1:10002/ws"
```

Comment:
We do no longer need these?

Reply:
[Resolved](https://github.com/0xPolygon/polygon-edge/pull/386/commits/585d357ff9ff834ac33cc2fd98a060539046f5dc)